src/jquery.jeditable.js

```diff
@@ -339,13 +340,21 @@
 
                 form.submit(function(e) {
 
+                    /* Do no submit. */
+                    e.preventDefault();
+                    e.stopPropagation();
+
+                    if (isSubmitting) {
```

Comment:
I don't get this part. Can you explain here why you added the `isSubmitting` var?

Reply:
As I explained above: I experienced double submits, `form.submit` was called twice, and this was my work-around. You do not have to include this if I am the only person experiencing this? I have been experiencing this problem for a long time, but it didn't really bother me (my data was just saved twice) (I noticed this in the logfiles but just ignored it), until I attached a callback and then it became a "visual" problem. I tried making a test-case for it, but I need to find a way how to capture the calling of the ajax-request.

Reply:
To be clear: the user did not do anything strange, just press enter, or click 'Ok' (to start the submit action). It occurs in chrome, ie and firefox. The only thing deviating from the default is I style my buttons using bootstrap, which should not cause it imho?